server/models/address.go

```diff
@@ -0,0 +1,22 @@
+package models
+
+import "time"
+
+type Address struct {
+	Address                      string    `json:"address" bson:"address"`
+	Owner                        string    `json:"owner,omitempty" bson:"owner"`
+	Balance                      int64     `json:"balance" bson:"balance"` //backward compability
```

Comment:
This field comes through as a string fractional in the current version: `"balance":"18859853.03232"`